src/test/scala/com/snowflake/snowpark_test/OpenTelemetrySuite.scala

```diff
@@ -366,4 +440,15 @@ class OpenTelemetrySuite extends OpenTelemetryEnabled {
   }
 
   private val stageName1 = randomName()
+
+  def checkSpan(className: String, funcName: String, methodChain: String): Unit = {
+    val stack = Thread.currentThread().getStackTrace
```

Comment:
I improved this function, now, it can automatically detect line numbers in tests.